### PR TITLE
chore: initial project

### PR DESCRIPTION
**Pull request by galta95**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM node:12.18.3-slim as node-base
+RUN apt-get update && apt-get -y install wget
+RUN mkdir /confd
+RUN wget -O '/confd/confd' 'https://github.com/kelseyhightower/confd/releases/download/v0.15.0/confd-0.15.0-linux-amd64'
+RUN chmod +x /confd/confd
```
Comment (syncush) [inline Dockerfile:5]: lets drop confd

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+COPY ./package*.json ./
+RUN npm install --only=production
+COPY ./docs ./docs
+COPY ./confd ./confd
+COPY --from=build /usr/src/app/dist .
+
+RUN chown -R app . && mkdir -p /home/app/.config && chown -R app:app /home/app
+USER app:app
+
+HEALTHCHECK CMD wget http://127.0.0.1:${SERVER_PORT}/liveness -O /dev/null || exit 1
```
Comment (syncush) [inline Dockerfile:31]: drop

File: commitlint.config.js [hunk 3]
```diff
@@ -0,0 +1 @@
+module.exports = { extends: ['@commitlint/config-conventional'] };
```
Comment (syncush) [inline commitlint.config.js:1]: let's add the relevant scopes

**Review by syncush — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (galta95) [reply Dockerfile:5]: done

File: Dockerfile (hunk 2 above)
Comment (galta95) [reply Dockerfile:31]: done

File: commitlint.config.js (hunk 3 above)
Comment (galta95) [reply commitlint.config.js:1]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**